### PR TITLE
Docker image for the environment

### PR DESCRIPTION
**Pull request by dmiszkiewicz**

### CODE REVIEW

File: tutorial/index.html [hunk 1]
```diff
@@ -70,38 +70,26 @@
...
+    <br />
+        <code>cd /where/this/activator/template/is/located/<br />
+            psql -h localhost -U postgres -f ./postgres/init.sql &&<br />
+            psql -h localhost -U postgres -f ./postgres/auth_entry.sql &&<br />
+            psql -h localhost -U postgres -f ./postgres/identity.sql</code>
     </p>
 
     <h3>Running</h3>
-    <p>Before starting anything make sure you have PostgreSQL, MongoDB and Redis up and running.</p>
+    <p>Run <code>docker-compose</code> up in project main directory to lunch databases, or if you are using your own database instances, make sure you have PostgreSQL, MongoDB and Redis up and running.</p>
```
Comment (luksow) [inline tutorial/index.html]: Typo, should be `launch`.

File: tutorial/index.html [hunk 2]
```diff
@@ -70,38 +70,26 @@
...
 
     <h4>Akka HTTP</h4>
     <p>You can run each service separately, but we also we provided a SBT task called <code>runAll</code>.</p>
 
-    <h4>Play</h4>
+    <h4>Play</h4>psql identity_manager -U postgres -f ./identity-manager/src/main/resources/identity.sql
```
Comment (luksow) [inline tutorial/index.html]: Hmm, something's wrong here?

Comment (dmiszkiewicz) [inline tutorial/index.html]: Done!

File: tutorial/index.html (hunk 1 above)
Comment (dmiszkiewicz) [inline tutorial/index.html]: Done!

**Pull request merged**